src/Filesystem/Writer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SitemapPlugin\Filesystem;
+
+use Gaufrette\FilesystemInterface;
+
+final class Writer
+{
+    /** @var FilesystemInterface */
+    private $filesystem;
+
+    public function __construct(FilesystemInterface $filesystem)
+    {
+        $this->filesystem = $filesystem;
+    }
+
+    public function write(string $path, string $contents): void
+    {
+        $this->filesystem->write($path, $contents, $overwrite = true);
```

Comment:
As mentioned on Slack the problem with Gaufrette is its inability to write to streams. This is a problem with larger sitemaps

Reply:
This is possible with flysystem: https://flysystem.thephpleague.com/docs/usage/filesystem-api/#write-files-using-a-stream

Reply:
I doubt it, as Gaufrette also has a `public function createStream($key);` method defined. If possible I'll stick to Gaufrette as it's the Sylius default. Streams will be handled in a separate PR, else this PR becomes too big.

Reply:
https://github.com/stefandoorn/sitemap-plugin/issues/100

Reply:
Yes, that's cool :)